source/features/closing-remarks.tsx

```diff
@@ -61,14 +73,14 @@ function addExistingTagLink(tagName: string): void {
 		);
 	}
 
+	const linkedTag = <a href={tagUrl} className="text-mono text-small">{tagName}</a>;
 	attachElement('#issue-comment-box', {
 		before: () => (
 			<TimelineItem>
 				{createBanner({
-					text: <>This pull request first appeared in <span className="text-mono text-small">{tagName}</span></>,
-					classes: ['flash-success'],
-					action: tagUrl,
-					buttonLabel: <><TagIcon/> See release</>,
```

Comment:
This widget only shows the _state_ of the PR, it doesn't require any actions, so I think it'd make sense to _not_ use the button.

This now also helps differentiate the two states:

- released
- unreleased

… because they used to be clearly blue and clearly green, but now their color varies very little.